app/components/Dashboard/AssetBalancesPanel/AssetBalancesPanel.jsx

```diff
@@ -99,11 +107,35 @@ export default class AssetBalancesPanel extends React.Component<Props> {
     return neoPrice && NEO !== '0' ? neoPrice * toNumber(NEO) : 0
   }
 
+  getNEOFormattedPriceChange = (): string => {
+    return (this.props.neoPriceChange >= 0 ? '+' : '') + (this.props.neoPriceChange * 100).toFixed(2) + '%'
+  }
+
+  getNEOPriceChangeDirection = (): string => {
```

Comment:
I think you can specify the type here

`type PriceDirection = 'increase' | 'decrease'`

`getNEOPriceChangeDirection = (): PriceDirection`